tools/cla_check_in_travis.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -ev
+
+email="$(git log -1 $TRAVIS_COMMIT --pretty="%aE")"
+echo "Checking the licence agreement for $email..."
```

Comment:
@kyrofa I copied it from the website. But now I see it's used inconsistently, so I reported a bug:
https://github.com/ubuntudesign/www.ubuntu.com/issues/1200

I think we should use licence, even though it looks ugly.